src/app/functions/client/deleteAllCookies.ts

```diff
@@ -2,10 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { Cookies } from "react-cookie";
+
 export function deleteAllCookies() {
-  const cookieParts = document.cookie.split(";");
-  const cookieNames = cookieParts.map((part) => part.trim().split("=")[0]);
-  cookieNames.forEach((cookieName) => {
-    document.cookie = `${cookieName}=; expires=Thu, 01 Jan 1970 00:00:00 GMT`;
-  });
+  const cookies = new Cookies(null, { path: "/" });
+  Object.keys(cookies.getAll()).forEach((cookieName) =>
+    cookies.remove(cookieName),
+  );
```

Comment:
This is a nice improvement :) Modifying `document.cookie` directly as a string always makes me a little nervous.